src/components/InputTodo.js

```diff
@@ -1,3 +1,35 @@
-const InputTodo = () => <div>input field here...</div>;
+/* eslint-disable react/prop-types */
+import { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+
+const InputTodo = ({ addTodoItem }) => {
+  const [title, setTitle] = useState('');
+  const [message, setMessage] = useState('');
+  const handleChange = (e) => {
+    setTitle(e.target.value);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim()) {
+      addTodoItem(title);
+      setTitle('');
+      setMessage('');
+    } else {
+      setMessage('Please add an item');
+    }
+  };
+  return (
+    <>
+      <form className="form-todo" onSubmit={handleSubmit}>
+        <input type="text" className="form-todo-input" placeholder="Add todo..." value={title} onChange={handleChange} />
+
+        <button type="submit" className="todo-btn">
+          <FaPlusCircle />
+        </button>
```

Comment:
Your application is not functioning as intended, the input field and submit button are not showing on the live version, thus unable to Create, Read, Update and Delete (CRUD). Please be so kind and rectify.